package/network/services/openvpn/files/openvpn.config

```diff
@@ -398,7 +398,7 @@ config openvpn sample_client
 	# LZ4 requires OpenVPN 2.4+ on server and client
 #	option compress lz4
 	# LZO is compatible with most OpenVPN versions
-	option compress lzo
+#	option compress lzo
```

Comment:
So we don't have ANY compression modes?  Why is that?

Reply:
https://serverfault.com/a/755690
https://arstechnica.com/civis/viewtopic.php?p=1307910#p1307910
https://www.snbforums.com/threads/openvpn-server-settings-connection-very-slow.37998/#post-313327
etc
Pretty much similar to my experience on various platforms.